protobuf-codegen/src/message.rs

```diff
@@ -347,6 +347,10 @@ impl<'a> MessageGen<'a> {
         self.fields.len() <= 500
     }
 
+    fn supports_serde(&self) -> bool {
+        self.customize.serde_derive.is_some() && self.customize.serde_derive.unwrap()
```

Comment:
This could be written as `self.customize.serde_derive.unwrap_or(false)`.

Could you please inline this function `supports_serde` and put this condition near `derive` generation?

Reply:
Inlined it - but now I'm realizing that I'll need to call this again when conditionally adding `#[serde(ignore)]` so I'm thinking we keep this in a function?

Reply:
OK, but call it something different then, like `serde_derive_enabled` or `enabled_serde_derive` then.

`supports_derive_partial_eq` is different, it is not about querying options, it is about work around rustc bug.

Reply:
Awesome - thanks - renamed